geotrek/trekking/models.py

```diff
@@ -607,6 +608,25 @@ def save(self, *args, **kwargs):
         super(DifficultyLevel, self).save(*args, **kwargs)
 
 
+class LabelTrek(PictogramMixin):
+
+    name = models.CharField(verbose_name=_("Name"), max_length=128)
+    description = models.TextField(verbose_name=_("Description"), blank=True,
+                                   help_text=_("Description of the label"), default='')
+    advice = models.TextField(verbose_name=_("Advices"), blank=True,
+                              help_text=_("Advice linked with the label"), default='')
```

Comment:
On dit "linked to" https://www.usingenglish.com/forum/threads/108104-LINKED-TO-or-LINKED-WITH

(j'ai relu les traductions avant, désolée)